### DISCUSSION TITLE
does not run on intel arc

### DISCUSSION THREAD

**Discussion by kosadko** [vladmandic/sdnext#3543] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/250314ea-d66d-4023-8a0a-ed527c11bf4d)
Good afternoon. I recently decided to reinstall sdnext and ran into this problem. Before that, everything worked and started normally. My pc: AMD Ryzen 5 5500, Intel arc a750. To be honest, my English is not very good and my attempts to find a solution have not been successful

**Comment by Disty0** [comment, marked as answer]
Duplicate of https://github.com/vladmandic/automatic/issues/3548.

Install OneAPI and run these in CMD, then start the webui within that CMD window.

```
"C:\Program Files (x86)\Intel\oneAPI\pytorch-gpu-dev-0.5\oneapi-vars.bat"
"C:\Program Files (x86)\Intel\oneAPI\ocloc\2024.2\env\vars.bat" 
```
Then;
```
.\webui.bat --use-ipex
```

Updated the Wiki to include these as well.

**Comment by Disty0** [comment]
More info about these changes:

We switched to PyTorch 2.6 XPU nightlies on Windows for Intel because Intel forces Anaconda to use IPEX on Windows and last usable version of IPEX that doesn't force Anaconda has its own issues (like being unable to generate 1024x1024 sized resolutions and causing random corruptions).

PyTorch 2.6 XPU doesn't force Anaconda but it seems like Intel choosed to force OneAPI as an extra step instead this time.
